scripts/publish/bintray-debian.sh

```diff
@@ -37,29 +71,25 @@ if [ -z "$BINTRAY_USER" ] || [ -z "$BINTRAY_API_KEY" ]; then
 fi
 set -u
 
-ARGV_FILE=$1
-PACKAGE_COMPONENT=etcher
-PACKAGE_DISTRIBUTION=stable
-PACKAGE_FILE_NAME=$(basename $ARGV_FILE)
-PACKAGE_NAME=${PACKAGE_FILE_NAME%.*}
-PACKAGE_VERSION=$(echo $PACKAGE_NAME | cut -d_ -f2 | tr "~" "-")
-PACKAGE_ARCH=$(echo $PACKAGE_NAME | cut -d_ -f3)
-
-if [ -z $PACKAGE_VERSION ]; then
-  echo "Couldn't infer the version from the package file name" 1>&2
+if [ "$ARGV_RELEASE_TYPE" == "production" ]; then
+  PACKAGE_DISTRIBUTION=stable
+elif [ "$ARGV_RELEASE_TYPE" == "snapshot" ]; then
+  PACKAGE_DISTRIBUTION=devel
+else
+  echo "Invalid release type: $ARGV_RELEASE_TYPE" 1>&2
   exit 1
 fi
 
-if [ -z $PACKAGE_ARCH ]; then
-  echo "Couldn't infer the architecture from the package file name" 1>&2
-  exit 1
-fi
+PACKAGE_COMPONENT=$(echo "$ARGV_APPLICATION_NAME" | tr '[:upper:]' '[:lower:]')
```

Comment:
Rather than doing the `tr` here, it's probably better to just pass in `$(APPLICATION_NAME_LOWERCASE)` from the Makefile?

And I guess you _could_ pass in `$(APPLICATION_VERSION_DEBIAN)` too?